### PR TITLE
Remove link to Dev UI v1 as it is now gone

### LINKED ISSUE #36109: Creating an app via code.quarkus.io still advertises dev-v1 in the welcome page
### Describe the bug

When running the downloaded app via quarkus:dev then browsing to localhost:8080

the following can be seen where /q/dev-v1 will return a 404;
Screen Shot 2023-09-22 at 2 29 33 PM

### Expected behavior

Link should point to /dev or /dev-ui

### Actual behavior

points to /dev-v1 which is invalid/removed

### How to Reproduce?

Download an app form code.quarkus.io and run in in dev mode then browse to http://localhost:8080

### Output of `uname -a` or `ver`

_No response_

### Output of `java -version`

_No response_

### GraalVM version (if different from Java)

_No response_

### Quarkus version or git rev

3.4.1

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Sap004**
fixes #36109

### CODE REVIEW

File: independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus/project/quarkus/base/src/main/resources/META-INF/resources/index.tpl.qute.html [hunk 1]
```diff
@@ -255,7 +255,7 @@ <h1>You just made a Quarkus application.</h1>
                     <p>App configuration: <code>src/main/resources/{config.file-name}</code></p>
                     <p>Static assets: <code>src/main/resources/META-INF/resources/</code></p>
                     <p>Code: <code>{language.dir.code}</code></p>
-                    <p>Dev UI V1: <a href="/q/dev-v1/">/q/dev-v1</a></p>
+                    <p>Dev UI V1: <a href="/q/dev-ui/">/q/dev-v1</a></p>
```
Comment (manofthepeace) [inline independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus/project/quarkus/base/src/main/resources/META-INF/resources/index.tpl.qute.html:258]: V1  should be removed from the '\<p\>' as well

Comment (Sap004) [reply independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus/project/quarkus/base/src/main/resources/META-INF/resources/index.tpl.qute.html]: hi @manofthepeace ,
Removed V1 from <p>, requesting for approval

**Review by phillip-kruger — APPROVED** [review]
Thanks !

File: independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus/project/quarkus/base/src/main/resources/META-INF/resources/index.tpl.qute.html [hunk 2]
```diff
@@ -255,7 +255,7 @@ <h1>You just made a Quarkus application.</h1>
                     <p>App configuration: <code>src/main/resources/{config.file-name}</code></p>
                     <p>Static assets: <code>src/main/resources/META-INF/resources/</code></p>
                     <p>Code: <code>{language.dir.code}</code></p>
-                    <p>Dev UI V1: <a href="/q/dev-v1/">/q/dev-v1</a></p>
+                    <p>Dev UI: <a href="/q/dev-ui/">/q/dev-ui</a></p>
```
Comment (gsmet) [inline independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus/project/quarkus/base/src/main/resources/META-INF/resources/index.tpl.qute.html:258]: The Dev UI link is already present above. Let's drop this altogether. I will fine tune this and get this puppy home.

**Review by gsmet — APPROVED** [review]
This should be good to go now. Let's wait for CI and merge.

**Pull request merged**